uwrt_mars_rover_utils/uwrt_mars_rover_xbox_controller/launch/location_launch.py

```diff
@@ -0,0 +1,22 @@
+import launch
```

Comment:
I think you should just remove this launch file and add your composable node into the container I made below because your node depends on my xbox_node to function.